debug_module.tex

```diff
@@ -108,7 +108,7 @@ \section{Reset Control} \label{reset}
 These bits must be set regardless of the cause of the reset.
 The {\tt havereset} bits for the selected harts
 can be cleared by writing 1 to \FdmDmcontrolAckhavereset in \RdmDmcontrol.
-The {\tt havereset} bits may or may not be cleared
+The {\tt havereset} bits may be cleared
```

Comment:
To me, the "may or may not" language seems clearer. Perhaps "might" or "might or might not"?

Reply:
OK.  I was going with IEEE terminology but that's basically not followed anywhere else so there's no point in doing that here.  I just pushed a change that follows your suggestion.